lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -10,9 +10,10 @@ def name
       # need: params['percentage']
       def is_enabled?(params = {}, context = nil)
         return false unless params.is_a?(Hash)
-        return false unless context.instance_of?(Unleash::Context)
 
         stickiness = params.fetch('stickiness', 'default')
+        return false unless context.instance_of?(Unleash::Context) || ['random', 'default'].include?(stickiness)
```

Comment:
```suggestion
        return false if is_context_invalid?(context, stickiness)
```

```ruby
# Context is always valid when using random or default strategies.
def is_context_invalid?(context, stickiness)
    context.instance_of?(Unleash::Context) || ['random', 'default'].include?(stickiness)
end
```

Reply:
Feel free to rename the method, not sure it's the most appropriate. Just a suggestion.

Reply:
I did it slightly different in order to keep the argument order and be more explicit.